Specs/Scene/BatchTextureSpec.js

```diff
@@ -10,6 +10,7 @@ describe(
     const mockTileset = {
       _statistics: {
         batchTableByteLength: 0,
+        metadataByteLength: 0,
```

Comment:
this field wasn't actually needed. Remove it and sweep through other files as well.